app/models/miq_worker.rb

```diff
@@ -383,39 +341,25 @@ def systemd_worker?
   end
 
   def start_runner
-    if ENV['MIQ_SPAWN_WORKERS'] || !Process.respond_to?(:fork)
-      start_runner_via_spawn
-    elsif systemd_worker?
+    if ENV['MIQ_SYSTEMD_WORKERS'] && systemd_worker?
```

Comment:
@agrare @carbonin feel free to suggest a better name.  Before spawn was an option, now, it's the default so I needed a way to opt-in to using systemd workers instead.

Reply:
:+1: that's good name